examples/launch/launch-quickstart/train.py

```diff
@@ -76,7 +76,7 @@ def forward(self, x):
       losses.append(loss.item())
 
       if iter % 25 == 1:
-        print("loggin")
+        print("logging")
```

Comment:
IMO we should remove this and the print(iter) above

Reply:
Ooph, no kidding. I can do that.